index.html

```diff
@@ -5,45 +5,364 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <title>Week 1: HTML Tables</title><!-- Latest compiled and minified CSS -->
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+   <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Lato">
+
   <style>
     .header {
-      text-align: center;
+      text-align: left;
+      margin-top: 70px;
+     
     }
+
+    .income {
+      font-color: #F2998F;
+    }
+
+    .bar {
+          height: 20px;
+          background: #F2998F;
+        }
+      
+    .pager>li.previous>a {
+      color: #000000;
+    }
+
+    .pager>li.next>a {
+      color: #000000;
+    }
+
+   table {
+      color:  #ffffff;
+      margin-top: 50px;
+   
+    }
+
+    th {
+      font-size: 16px;
+    }
+
+    body {
+      background: #357C8D;
+      color: #ffffff;
+      font-family: "Lato";
+
+    }
+
+    h1 {
+      font-family: "Lato";
+      font-size: 30px;
+    }
+    img {margin: 50px 50px;}
+  
+    h2 {
+      font-family: "Lato";
+      font-size: 20px;
+      
+    }
+
+    .table-hover tbody tr:hover td {
+      background-color: #5a5858;
+    
+    }
+
+    a {
+      color: #ffffff;
+
+    }
+
+    .btn.outline {
+      background: none;
+      padding: 6px 6px;
+      margin: 8px 8px;
+
+    }
+
+    .btn-primary.outline {
+      border: 2px solid #ffffff;
+      color: #ffffff;
+    
+    } 
+    
+    .btn-primary:hover {
+      color: #fff;
+    background-color: #5a5858;
+    border-color: #ffffff; 
+    }
+ 
+   
+
   </style>
 </head>
 <body>
   <div class="container">
     <div class="header">
-      <h1>Week 1: HTML Tables</h1>
-      <p>We'll add our data to this table.</p>
+      <div class = "col-sm-7">
+        <h1>A Glance at D.C. Neighborhoods</h1>
+        <br />
```

Comment:
Let's try and avoid using `<br>` elements. We can do this better with just CSS, for example by setting a `margin-bottom` on the `h1`.